scripts/docker_setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export PATH=/opt/python/cp37-cp37m/bin:$PATH
+pip install cmake==3.17.3
+cmake --version
+which g++
+which python
```

Comment:
Is this docker script used anywhere? I can't find it being used.

Reply:
I'm forked a github action and changed the container so it would do what I needed.
it is:
https://github.com/allen-cell-animated/python-wheels-manylinux-build

Reply:
I guess I am confused even more now. If this file is for a different repo why is it in this repo?

Reply:
This one isn't it was initially and helped me debug. echo stuff check paths check version numbers etc. 
Once I got it working it wasn't needed but I left it in the repo. Should I nuke it?

Reply:
If you think you are going to use it in the future I would keep it but place a comment at the top of the file explaining what it is used for. If you don't plan on using it in the future I would nuke it.

Reply:
totally agree. if unused, nuke it. :)